src/main/java/com/minecolonies/coremod/items/ItemScepterLumberjack.java

```diff
@@ -107,22 +101,38 @@ private void storeRestrictedArea(final PlayerEntity player, final Hand hand, fin
             return;
```

Comment:
Can't comment on lines that are not in this commit, but maxArea was changed in the older PR